help/deploy/releases/release-history.md

```diff
@@ -20,155 +20,174 @@ position: 6 # float position is supported
-* (Improvement) GROK-16215: JS API: Force DG.FileInfo to have `name` 
-* (Bug) GROK-16216: Inputs: FileInput does not trigger onInput (WIP)
-* GROK-15922: EDA : Linear Regression 
-* (Bug) [#2924](https://github.com/datagrok-ai/public/issues/2924): #2924: Curves: connectDots doesn't work if fit is turned off 
-* (Bug) GROK-15798: Bio: Fix To Atomic Level for units FASTA, UN alphabet. (WIP)
-* (Improvement) GROK-16240: JS API: Replace metadata set by .tags with column.meta.xxx 
-* (Bug) GROK-16238: Docker: Failed package deploy with docker config 
-* GROK-14293: Core: Inputs harmonization (WIP)
-* GROK-16242: Alexander & Govern's test implementation 
-* GROK-15413: Sharepoint data provider 
-* (Bug) GROK-16223: Test results dataset won't open 
-* (Bug) GROK-16222: Browse: Files don't open if a search field is not empty 
-* (Bug) GROK-16244: Box plot | Violin style: category bug 
-* (Improvement) GROK-16247: Core: Tooltips: ability to click a link in the tooltip 
-* (Bug) GROK-16248: Proxy storage writeFileStream method not implemented 
-* GROK-14130: Decorators (WIP)
-* (Bug) GROK-16234: Bar chart: wrong label shortening 
-* (Bug) GROK-16235: #2925: Viewer title is not saved in layout if it was edited in viewer's header 
-* (Improvement) GROK-16237: #2926: Heatmap: save scrollbar position for layout 
-* (Bug) GROK-15793: Bio: Calculate Identity, Similarity throw Index out of bounds (WIP)
-* (Bug) GROK-16269: History prints wrong timestamps 
-* (Improvement) GROK-16274: Prevent auto running of functions in browse preview (WIP)
-* (Bug) GROK-15490: Fix UI issues with scaling on different browser sizes (WIP)
-* (Improvement) GROK-16212: Auto tests creation: sharing \- objects in general, and file shares and db connectios 
-* (Bug) GROK-15967: Context panel | When you share the connection \- the information doesn`t update (WIP)
-* (Bug) GROK-15324: Widgets | Recent projects: context menu fixing 
-* (Bug) GROK-16228: JS API: `grok.functions.calls.list` returns incorrect Func's nqName functions 
-* (Improvement) GROK-11494: Widgets: events (WIP)
-* (Improvement) GROK-16278: Avoid sending extra data in files serialization 
-* (Improvement) GROK-16279: Make browse less chatty (WIP)
-* (Bug) GROK-16275: #2771: Viewers: filter out legend with empty categories in the legend 
-* (Bug) GROK-16277: #2934: (Mac OS) Grid: some hotkeys for selection/deselection are not working for columns 
-* (Improvement) GROK-16254: #2932: Viewers: hide the option “Is Grid” from property panel 
-* (Bug) GROK-16320: Predictive modeling: interactive model saved without a name (WIP)
-* (Bug) GROK-16309: Set tooltip dialog looks bad 
-* (Bug) GROK-16331: Core: Bar Chart: molecules are renderered incorrectly on resizing bar chart 
-* (Bug) GROK-16330: Core: Table view: Form styles with refresh button is broken 
 
+## 2024-07-23 Datagrok 1.20.0 release 
+ 
+### Breaking changes:
+ 
+* Lightweight Predictive modeling improvements.
+* File caching updates.
```

Comment:
@LesiaPavlenko How are these breaking changes? I may be wrong but the list here is different from what we have in the compatibility tool for the same

Reply:
Agree, I'll move them to "Improvements" part